bugsnag-android-performance/src/main/kotlin/com/bugsnag/android/performance/LoadingIndicatorView.kt

```diff
@@ -0,0 +1,36 @@
+package com.bugsnag.android.performance
+
+import android.content.Context
+import android.util.AttributeSet
+import android.widget.FrameLayout
+import com.bugsnag.android.performance.internal.BugsnagPerformanceInternals
+import com.bugsnag.android.performance.internal.SpanCategory
+import com.bugsnag.android.performance.internal.SpanContextStack
+import com.bugsnag.android.performance.internal.SpanImpl
+import com.bugsnag.android.performance.internal.SpanImpl.Condition
+
+private const val CONDITION_TIMEOUT = 100L
+
+public class LoadingIndicatorView @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyleAttr: Int = 0,
+) : FrameLayout(context, attrs, defStyleAttr) {
+    internal var condition: Condition?
```

Comment:
this should be `private`
```suggestion
    private var condition: Condition?
```